### ISSUE TITLE
Cantherm header needs updating

### ISSUE THREAD

**Issue by rwest** [ReactionMechanismGenerator/RMG-Py#828]
When you run a Cantherm job on the official 2.0.0 build, the log file begins:
```
###############################################################
#                                                             #
#                          CanTherm                           #
#                                                             #
#   Version: 0.1.0 (14 May 2009)                              #
#   Authors: RMG Developers (rmg_dev@mit.edu)                 #
#   P.I.:    William H. Green (whgreen@mit.edu)               #
#   Website: http://rmg.sourceforge.net/                      #
#                                                             #
###############################################################
```

The version, date, and website are all wrong.

Maybe it should also include information on how to cite it properly.

**Comment by KEHANG** [comment]
will be fixed by PR #846 

As for the citation, I'll bring it up in RMG meeting next week but if you have any suggestions please let me know as well.

**Comment by mliu49** [comment]
#846 has been merged into master.
